app/assets/stylesheets/styles.scss

```diff
@@ -22,6 +21,15 @@ $bandtree-green: #43ac6a;
   text-align: center;
 }
 
+.splash-text {
+  background-color: #222222;
```

Comment:
Color literals like `#222222` should only be used in variable declarations; they should be referred to via variable everywhere else.